Update To Pm5

GabBiswajit opened the pull request:
I Don't Test The Plugin But MaBey It's Work.

GabBiswajit committed:
Update To Pm5

GabBiswajit committed:
Update README.md

GabBiswajit committed:
Update To Pm5

GabBiswajit committed:
Update To Pm5

GabBiswajit committed:
Update to pm5

GabBiswajit committed:
Update To Pm5

note3crafter commented:
i have a privat build sure thats all features works fine? can you make a vid for youtube?

GabBiswajit commented:
Ok I make tomorrow

GabBiswajit commented:
I think it's work if not than I will fix it again in few minutes.

note3crafter commented:
okidoki

GabBiswajit commented:
@note3crafter it's perfectly work I use this on my server !

GabBiswajit commented:
But i don't understand how i use /portal flag addcommand "my command" it's not work

note3crafter commented:
yea thats the problem... command section is broken

GabBiswajit commented:
Hmm I see

poggit-bot committed:
Create .poggit.yml
Poggit-CI is enabled for this repo by @GabBiswajit
Visit the Poggit-CI page for this repo at https://poggit.pmmp.io/ci/GabBiswajit/PortalsPE-PM5

poggit-bot committed:
Create .poggit.yml
Poggit-CI is enabled for this repo by @GabBiswajit
Visit the Poggit-CI page for this repo at https://poggit.pmmp.io/ci/GabBiswajit/PortalsPE-PM5
